src/styles/main.scss

```diff
@@ -1,3 +1,47 @@
+$cell-size:100px;
+$week-day: "mon" 1, "tue" 2, "wed" 3, "thu" 4, "fri" 5, "sat" 6, "sun" 7;
+
 body {
   margin: 0;
 }
+
+.calendar {
+  display: flex;
+  margin: auto;
+  flex-wrap: wrap;
+  width: 720px;
+  gap: 1px;
+  border: 1px solid black;
+
+  &__day {
+    display: flex;
+    width: $cell-size;
+    height: $cell-size;
+    background-color: #eee;
+    border: 1px solid black;
+    &:hover {
+      cursor: pointer;
```

Comment:
let's move it to `&_day`

Reply:
didn't catch this
It already belongs to $__day
Could you pls explain me?

Reply:
```
&_day{
cursor: pointer;
...
 &:hover{...}
}
```